main.tf

```diff
@@ -41,6 +41,16 @@ resource "google_dns_record_set" "a" {
   rrdatas = [google_compute_global_address.static_ip[0].address]
 }
 
+resource "google_dns_record_set" "a_apps" {
+  count        = var.google_dns_managed_zone.enabled ? 1 : 0
+  name         = "apps-${var.deploy_id}.${var.google_dns_managed_zone.dns_name}"
```

Comment:
why is this hardcoded vs a prefix like our config?
we dont need a caa for the new A record?

Reply:
I can make it configurable or move it internally, fair enough

> we dont need a caa for the new A record?
good question, it does appear to work, but maybe it's just much slower?

```
$ dig CAA apps-stevedgkoa.eng-platform-dev.domino.tech

; <<>> DiG 9.10.6 <<>> CAA apps-stevedgkoa.eng-platform-dev.domino.tech
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 1728
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;apps-stevedgkoa.eng-platform-dev.domino.tech. IN CAA

;; AUTHORITY SECTION:
eng-platform-dev.domino.tech. 300 IN	SOA	ns-cloud-e1.googledomains.com. cloud-dns-hostmaster.google.com. 1 21600 3600 259200 300

;; Query time: 55 msec
;; SERVER: 192.168.50.1#53(192.168.50.1)
;; WHEN: Wed Jan 24 16:02:18 PST 2024
;; MSG SIZE  rcvd: 169

$ echo | openssl s_client -connect apps-stevedgkoa.eng-platform-dev.domino.tech:443 |& grep Verification
Verification: OK
```

Reply:
It might be a Google Trust vs. LetsEncrypt thing. Either way safer to add the CAA.